### ISSUE TITLE
[RFE]: provide solutions

### ISSUE THREAD

**Issue by notklaatu** [SELinuxGame/selinuxgame.org#16]
This is an idea, and it clashes with your existing leaderboard idea, but I'm mentioning it anyway.

If the intent is to teach users SELinux, then there ought to be a command to reveal the solution within each scenario. The subtitle of the site is "Learn SELinux by doing", and although a user may try to do SELinux stuff, but without solutions, then the learning isn't guaranteed (indeed, the user might just learn the wrong things, or draw incorrect conclusions).

It's not realistic to rely on live admins in IRC to provide answers, especially since the scenarios can be run offline in an environment where Internet access is unavailable; a student might progress through several scenarios before having a chance to get to an IRC admin.

tl;dr: make the scenarios self-contained, with solutions and explanations to the problems.
